### ISSUE TITLE
Add title on search input for accessbility

### ISSUE THREAD

**Issue by bdurand** [bluzky/nice-select2#32]
The search box on searchable menus does not have a label and is flagged by some accessibility audits. Adding a title attribute to the input element would make the feature more accessible for screen readers.
